lms/djangoapps/discussion/templates/discussion/discussion_board_fragment.html

```diff
@@ -14,42 +12,11 @@
 
 from django_comment_client.permissions import has_permission
 from openedx.core.djangolib.js_utils import dump_js_escaped_json, js_escaped_string
+from openedx.core.djangolib.markup import HTML
 %>
 
-<%block name="bodyclass">discussion</%block>
-<%block name="pagetitle">${_("Discussion - {course_number}").format(course_number=course.display_number_with_default)}</%block>
-
-<%block name="headextra">
-<%include file="../discussion/_js_head_dependencies.html" />
-</%block>
-
-<%block name="base_js_dependencies">
-    ## Enable fast preview to fix discussion MathJax rendering bug when page first loads.
-    <%include file="/discussion/_js_body_dependencies.html" args="disable_fast_preview=False"/>
-</%block>
-
-<%block name="js_extra">
-<%static:require_module module_name="discussion/js/discussion_board_factory" class_name="DiscussionBoardFactory">
-DiscussionBoardFactory({
-    courseId: '${unicode(course.id) | n, js_escaped_string}',
-    $el: $(".discussion-board"),
-    user_info: ${user_info | n, dump_js_escaped_json},
-    roles: ${roles | n, dump_js_escaped_json},
-    sort_preference: '${sort_preference | n, js_escaped_string}',
-    threads: ${threads | n, dump_js_escaped_json},
-    thread_pages: '${thread_pages | n, js_escaped_string}',
-    content_info: ${annotated_content_info | n, dump_js_escaped_json},
-    course_name: '${course.display_name_with_default | n, js_escaped_string}',
-    course_settings: ${course_settings | n, dump_js_escaped_json}
-});
-</%static:require_module>
-</%block>
-
-<%include file="../courseware/course_navigation.html" args="active_page='discussion'" />
-
-<%block name="content">
 <section class="discussion discussion-board container" id="discussion-container"
-         data-course-id="${course_id}"
+         data-course-id="${unicode(course.id)}"
```

Comment:
I removed the `course_id` parameter as it seemed unnecessary since the `course` was already being passed. I think I originally had `${course.id}` and something complained, but I'll try it again and see.